server/src/main/java/org/opensearch/search/builder/SearchSourceBuilder.java

```diff
@@ -340,6 +351,12 @@ public void writeTo(StreamOutput out) throws IOException {
         if (out.getVersion().onOrAfter(LegacyESVersion.V_7_10_0)) {
             out.writeOptionalWriteable(pointInTimeBuilder);
         }
+        if (out.getVersion().onOrAfter(Version.V_2_8_0)) {
```

Comment:
As always, we would need to bring it back to `main` :)

Reply:
Yup! 

I might piggy-back that into https://github.com/opensearch-project/OpenSearch/pull/7470